modules/ztp-validating-the-generation-of-configuration-policy-crs.adoc

```diff
@@ -80,31 +80,31 @@ ztp-site.example-sno-config-policy           inform               Compliant
 +
 {rh-rhacm} copies all applicable policies into the cluster namespace. The copied policy names have the format: `<policyGenTemplate.Namespace>.<policyGenTemplate.Name>-<policyName>`.
 
-. Check the placement rule for any policies not copied to the cluster namespace. The `matchSelector` in the `PlacementRule` for those policies should match labels on the `ManagedCluster` object:
+. Check the placement rule for any policies not copied to the cluster namespace. The `matchSelector` in the `{placement-rule-cr}` for those policies should match labels on the `ManagedCluster` object:
 +
 [source,terminal]
 ----
-$ oc get placementrule -n $NS
+$ oc get {placement-rule-cr} -n $NS
```

Comment:
I could see the code in the rendered page, is it missing quotes? Same for the 2 others below